src/components/EventDate.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement } from "react";
+
+// Renders an event's day of the month numerically, month, and day of the week.
+export const EventDate = (props: {
+  startTime: Date;
+  color: string;
+}): ReactElement => {
+  const date = {
+    num: props.startTime.getDate(),
+    month: props.startTime.toLocaleString("en-US", {
+      month: "short",
+    }),
+    day: props.startTime.toLocaleString("en-US", {
+      weekday: "short",
+    }),
+  };
+
+  const baseStyle =
+    "w-16 h-12 md:w-20 md:h-14 rounded-r-2xl flex justify-center items-center text-xl md:text-2xl font-bold text-white";
+
```

Comment:
this is very nitpicky but I feel like it might look cleaner to make another constant for the styling on the dates tab, so we don't have to do it all within the div.

`const tabStyling = baseStyle + props.color + (props.color === "bg-white" ? "text-black" : "text-white")`

lmk if this makes sense